features/gift_exchanges/challenge_yuletide.feature

```diff
@@ -248,7 +252,8 @@ Feature: Collection
   #     offers Tiny (Obscure, JS), S&H (Foo, JS)
   When I log out
     And I am logged in as "myname4"
-  When I go to the collections page
+  When all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.